From 4a4eeba63f141bc387520cbc4906b6bd338845a2 Mon Sep 17 00:00:00 2001
From: Soren Brinkmann <soren.brinkmann@xilinx.com>
Date: Thu, 10 Apr 2014 08:04:06 -0700
Subject: [PATCH 176/182] i2c: cadence: Update DT binding documentation

This patch comes from:
  https://github.com/Xilinx/linux-xlnx.git

Sync with mainline version.

Signed-off-by: Soren Brinkmann <soren.brinkmann@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
(cherry picked from commit 366e0397f2a3ba969191e07bf459380392051719)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 .../devicetree/bindings/i2c/i2c-cadence.txt        |   19 +++++++++++--------
 1 files changed, 11 insertions(+), 8 deletions(-)

diff --git a/Documentation/devicetree/bindings/i2c/i2c-cadence.txt b/Documentation/devicetree/bindings/i2c/i2c-cadence.txt
index 112e4a1..7cb0b56 100644
--- a/Documentation/devicetree/bindings/i2c/i2c-cadence.txt
+++ b/Documentation/devicetree/bindings/i2c/i2c-cadence.txt
@@ -1,20 +1,23 @@
 Binding for the Cadence I2C controller
 
 Required properties:
-  compatible: Compatibility string. Must be 'cdns,i2c-r1p10'.
-  clocks: From common clock bindings. Phandle to input clock.
+  - reg: Physical base address and size of the controller's register area.
+  - compatible: Compatibility string. Must be 'cdns,i2c-r1p10'.
+  - clocks: Input clock specifier. Refer to common clock bindings.
+  - interrupts: Interrupt specifier. Refer to interrupt bindings.
+  - #address-cells: Should be 1.
+  - #size-cells: Should be 0.
 
 Optional properties:
-  clock-frequency: Desired operating frequency, in Hz, of the bus.
+  - clock-frequency: Desired operating frequency, in Hz, of the bus.
+  - clock-names: Input clock name, should be 'pclk'.
 
 Example:
-
-	i2c0: cdns-i2c@e0004000 {
+	i2c@e0004000 {
 		compatible = "cdns,i2c-r1p10";
 		clocks = <&clkc 38>;
-		interrupt-parent = <&intc>;
-		interrupts = <0 25 4>;
-		reg = <0xE0004000 0x1000>;
+		interrupts = <GIC_SPI 25 IRQ_TYPE_LEVEL_HIGH>;
+		reg = <0xe0004000 0x1000>;
 		clock-frequency = <400000>;
 		#address-cells = <1>;
 		#size-cells = <0>;
-- 
1.7.5.4

